inc/namespace.php

```diff
@@ -161,6 +161,15 @@ function enable_lightbox_for_images( string $content ): string {
 		$content = $document_html;
 	}
 
+	// Remove stray end tags of void elements libxml2 does not know.
+	$stray_end_tags = [
+		'</embed>',
+		'</source>',
+		'</track>',
+		'</wbr>',
```

Comment:
Let's extend the existing `str_replace()` call with these tags, you can update the comment as well.